scipy/optimize/_highs/cython/src/HConst.pxd

```diff
@@ -6,43 +6,47 @@ from libcpp.string cimport string
 
 cdef extern from "HConst.h" nogil:
 
-    const int HIGHS_CONST_I_INF
-    const double HIGHS_CONST_INF
-    const double HIGHS_CONST_TINY
-    const double HIGHS_CONST_ZERO
-    const int HIGHS_THREAD_LIMIT
-    const bool allow_infinite_costs
-    const string FILENAME_DEFAULT
+    const int HIGHS_CONST_I_INF "kHighsIInf"
```

Comment:
The (private) Cython interface in most cases remains unchanged by using this pattern of "virtual typedefs".  HiGHS did a large refactor where all constants/enums where renamed to include a leading `k` and changing from underscores to camelCase and almost all changes to the PXD files reflect these naming changes